config.TEMPLATE.inc.php

```diff
@@ -575,6 +570,40 @@
 ; Remove this setting to leave failed jobs in the database.
 delete_failed_jobs_after = 180
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Schedule Task Settings ;
+;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+[schedule]
+
+; Whether or not to turn on the built-in schedule task runner
+;
+; When enabled, schedule tasks will be processed at the end of each web
+; request to the application.
+;
+; Use of the built-in schedule task runner is highly discouraged for high-volume 
+; sites. Use your operational system's task scheduler instead, and configure 
+; it to run the task scheduler every minute.
+; Sample for the *nix crontab
+; * * * * * php lib/pkp/tools/scheduler.php run >> /dev/null 2>&1
+;
+; See: <link-to-documentation>
+task_runner = On
+
+; How often should the built-in schedule task runner run scheduled tasks at the
+; end of web request life cycle (value defined the seconds).
```

Comment:
My suggestion was still incorrect 😁

```suggestion
; end of web request life cycle (value defined in seconds).
```